Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM node:6.1.0-onbuild
+FROM node:alpine
 
-RUN mkdir -p /verdaccio/storage /verdaccio/conf
+ENV appdir /usr/local/app
```

Comment:
By convention, environment variables (PAGER, EDITOR, ..) and internal shell variables (SHELL, BASH_VERSION, ..) are capitalized. All other variable names should be lower case. In this case i think it should be uppercase too.